src/plugins/data/public/index.ts

```diff
@@ -526,3 +526,5 @@ export {
   DataSourceGroup,
   DataSourceOption,
 } from './data_sources/datasource_selector';
+
+export { PersistedLog } from './query';
```

Comment:
can be a fast follow: we should add it to the other query: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/8b0af9de0215da61d23f4223bbce97a68d226e49/src/plugins/data/public/index.ts#L467